COMMERCE-9798 - defaultCPSubscriptionTypeLabel initial value is now the language key to 'day'

elocamp opened the pull request:
The problem was caused because the initial value of [defaultCPSubscriptionTypeLabel](https://github.com/liferay/liferay-portal/blob/8618a3437b6fca76e6f909e8aad4fe0b17da0649/modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/definition/edit_definition_subscription_info.jsp#L44) was being passed as blank.
Basically, the issue stops occurring when you select a Subscription Type.
Since the page always starts with Day as a timestamp in Subscription Length and the issue only occurs in the first view, the proposed fix is to add the language key to Day as initial value of defaultCPSubscriptionTypeLabel.

Related ticket: https://issues.liferay.com/browse/COMMERCE-9798

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

matthewchan123 commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest(master)/5339/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-3.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5339/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 24 out of 27 jobs passed in 2 hours 3 minutes 49 seconds 623 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/44db699cc5ead84479140190f245474199d4daba">44db699cc5ead84479140190f245474199d4daba</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/36a75fcae966182acf9988b97dd8069d75fa8b01">36a75fcae966182acf9988b97dd8069d75fa8b01</a></p>24 out of 27 jobs PASSED<div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest(master)/5339/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311171/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</strong></strike></a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265330/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</strong></strike></a></li></ul></div><details><summary><strong>24 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265327/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311173/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/301272/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265333/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/212780/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286577/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261379/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286574/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286573/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265329/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265332/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265331/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286575/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265328/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286576/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286572/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265326/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/174415/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286578/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286571/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/212779/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/174414/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/212781/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311172/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5339/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311171/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/311171//consoleText">AXIS_VARIABLE=0,label_exp=!master #311171</a><ol><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/311171//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmokeUpgrade_ViewPortalSmokeArchive704_">LocalFile.PortalSmokeUpgrade#ViewPortalSmokeArchive704</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-3/1562626894069/test-portal-acceptance-pullrequest(master)/5339/functional-upgrade-tomcat90-mysql57-jdk8/4/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive704/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-3/1562626894069/test-portal-acceptance-pullrequest(master)/5339/functional-upgrade-tomcat90-mysql57-jdk8/4/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive704/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-3/1562626894069/test-portal-acceptance-pullrequest(master)/5339/functional-upgrade-tomcat90-mysql57-jdk8/4/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265330/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/265330//consoleText">AXIS_VARIABLE=0,label_exp=!master #265330</a><div><p>Please fix <strong>semantic versioning</strong> on <strong><a href="https://github.com/matthewchan123/liferay-portal/tree/LPS-85535">matthewchan123/LPS-85535</a></strong><pre><code>    [javac] Note: Recompile with -Xlint:unchecked for details.

init-jar:

jar:
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-portal/portal-kernel/classes/META-INF/maven/com.liferay.portal/com.liferay.portal.kernel
[propertyfile] Creating new property file: /opt/dev/projects/github/liferay-portal/portal-kernel/classes/META-INF/maven/com.liferay.portal/com.liferay.portal.kernel/pom.properties
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null547690683.properties

download-latest-artifact:

init-compile:

compile:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null47650952.properties

download-latest-artifact:

init-compile:

compile:
     [copy] Copying 22 files to /opt/dev/projects/github/liferay-portal/util-java/classes
    [javac] Compiling 103 source files to /opt/dev/projects/github/liferay-portal/util-java/classes
    [javac] Note: /opt/dev/projects/github/liferay-portal/util-java/src/com/liferay/util/servlet/NullSession.java uses or overrides a deprecated API.
    [javac] Note: Recompile with -Xlint:deprecation for details.
    [javac] Note: Some input files use unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
 [sync-dir] Synchronizing /opt/dev/projects/github/liferay-binaries-cache-2017/.gradle/caches/modules-2/files-2.1 into /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1
 [sync-dir] 0 files synchronized in 458ms
     [exec] Using proxy squid.lax.liferay.com:3128.
     [exec] &gt; Task :classes UP-TO-DATE
     [exec] &gt; Task :jar SKIPPED
     [exec] 
     [exec] &gt; Task :baseline4
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec]   com.liferay.portal.kernel.service                  UNCHANGED  2.1.0      2.2.0      2.2.0      VERSION INCREASE SUGGESTED
     [exec] 	+   version    2.1.0
     [exec] 	-   version    2.2.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/portal-kernel/portal-kernel.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.kernel/4.22.0/71d2c10d40b6c32c1d9c1a99d9bb4cab58dba477/com.liferay.portal.kernel-4.22.0.jar</code></pre></p></div></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

brianikim commented:
Hello @matthewchan123 ,

Please fix the semantic versioning and resend.

Commit:
LPS-165169 Remove unused method stream()

jeffwu0724 committed:
LPS-169043 add commons-collections4 to source-formatter with version 4.4. We are using 4.2, 4.3, 4.4 in the system now, chose version 4.4, which is the highest version for better maintainance

Valmir-unicap committed:
LPS-166350 New Corrected the width to 25% and added the truncate an tooltip in text and select area.

Commit:
LPS-169456 ObjectDefintion The PUT method should only require the same fields as the POST method

Commit:
LPS-165169 Simplify, remove redundant method references in class SearchRequestContributorsRegistryImpl

Commit:
LPS-165169 Inline and remove unused classes

jeffwu0724 committed:
LPS-169043 export commons-collections4 from shared-dependencies-apache-commons with version 4.4

jeffwu0724 committed:
LPS-169043 auto sf

jeffwu0724 committed:
LPS-169043 manually change compileInclude to compileOnly for commons-collections4, because we can refer it from shared-dependencies-commons

jeffwu0724 committed:
LPS-169043 remove commons-collections4 from -includeresource in batch-engine-test, dynamic-data-lists-test and content-dashboard-test, because we can get the reference from shared-dependencies

Commit:
LPS-170771 Convert method reference to field reference(Because property ReferencePolicyOption.GREEDY exists, I did not directly reference AMImageEntryProcessor)

liuxingye committed:
LPS-166586 Add CanFilterEntriesUsingRelationshipField test

liuxingye committed:
LPS-166589 Add CanSeeEntriesWithFilterApplied test

aline-doleron committed:
LPS-166350 Do not show empty tooltip

javalosjr96 committed:
LPS-170857 Bump google.cloud:libraries-bom to 26.1.5

BND was also modified for new imports.

aline-doleron committed:
LPS-166350 Update snapshots

ces-totrinh committed:
LPS-171030 Elasticsearch7 adds field mapping for userGroupRoleIds

jackelyneaquino committed:
LPS-168479 Adding roles prior to permissions in order of invoke

LPS-168479 Assert Role Permission

LPS-168479 Assert role permission corrected

jeffwu0724 committed:
LPS-169043 auto generate, by running "ant build-lib-versions"

kevenleone committed:
LPS-171101 Create Dispatch Trigger Impl

ricardocousolr committed:
LPS-168169 Lower numerical value of priority means higher precendence

Entries with no priority (null) before are set to -1 in step v1_7_0, so
they have the most precendence. To respect this we must order by
priority asc too. Entries with the same priority value are sorted by
createDate (more recent means more precedence).

achaparro committed:
LPS-168721 Always show a message in the Upgrade Report

kevenleone committed:
LPS-171101 Update Object Definition

kevenleone committed:
LPS-171113 Call Dispatch Trigger after Create the Task

kevenleone committed:
LPS-171112 Update Status

kevenleone committed:
LPS-171112 SF

kevenleone committed:
LPS-171112 Enforce Type

andrea-ale-sbarra committed:
COMMERCE-10471 changed cookie type

evanilson-santana committed:
LPS-168858 Update groovy script with new object id name

evanilson-santana committed:
LPS-168858 add an expression on the edit task to validate the action was edited

tinatian committed:
LPS-165169 SF, simplify

CalumR23 committed:
POSHI-459 Generate CHANGELOG

CalumR23 committed:
POSHI-459 Update default poshi runner version

CalumR23 committed:
POSHI-459 Use latest gradle-plugins-poshi-runner

CalumR23 committed:
POSHI-459 Update poshi-core version

interaminense committed:
LRAC-12601 Fix commerce channels total when toggle commerce sync

Commit:
LRAC-12627 Modify the branch that is used in AC acceptance tests

interaminense committed:
LRAC-12601 Do no use uid because ids are generated every time table is rendered

interaminense committed:
LRAC-12601 Update generated types

kevenleone committed:
LPS-170826 Add Revalidate function

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Add Rest Implementations for Task, Subtask, CaseResult and TaskUsers

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Simplify

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Update Outlet parameters and standardize

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Use Outlet  with new properties on Testflow and Subtask

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Update Outlet Router

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

sandrodw3 committed:
LPS-170872 Sort

4lejandrito committed:
LPS-169781 Using local services

As in f6187cd26a98099ba86c801436cfa7eb0af961df that fixed this same
problem in the old dto converter.

victorg1991 committed:
LPS-171107 Remove unused role

brianchandotcom committed:
artifact:ignore frontend-taglib-util-freemarker-contributor 5.0.12 prep next

brianchandotcom committed:
POSHI-459 prep next

brianchandotcom committed:
POSHI-459 prep next

brianchandotcom committed:
POSHI-459 prep next

brianchandotcom committed:
Baseline

brianchandotcom committed:
LPS-168858 #sfme

Commit:
COMMERCE-9798 - defaultCPSubscriptionTypeLabel initial value is now the language key to 'day'

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

brianikim commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8618a3437b6fca76e6f909e8aad4fe0b17da0649">8618a3437b6fca76e6f909e8aad4fe0b17da0649</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/elocamp/liferay-portal/tree/COMMERCE-9798">COMMERCE-9798</a><br/>Branch GIT ID: <a href="https://github.com/elocamp/liferay-portal/commit/ad17980dcc9131b4122a2642b861113cc4389592">ad17980dcc9131b4122a2642b861113cc4389592</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-source-format/4874/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-source-format/builds/4874/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-7.liferay.com/job/test-portal-source-format/4874/">test-portal-source-format#4874</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-source-format/builds/4874/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/32">brianikim#32</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2136594933">[master] ci:test:sf - elocamp &gt; brianikim - PR#32 - 2022-12-21[14:09:20]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-7.liferay.com/job/publish-testray-report/2444/">publish-testray-report#2444</a><br/></div></div>

brianikim commented:
We will revisit this issue. I need to investigate the behavior. This will probably be done after my PTO.
